erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -278,7 +278,7 @@ def __str__(self):
 
     @staticmethod
     def get_eric_error_code_message(res_code):
-        return _ERIC_ERROR_MESSAGES.get(res_code, "Unknown error message")
+        return _ERIC_ERROR_MESSAGES.get(res_code, "UNKNOWN_ERROR")
```

Comment:
Good, following same capital-letters-pattern as we get from eric

Reply:
Yes